vanilla-framework/vanilla-framework
# Sync up design specs with scss components pt.1

## Done

- Updated icon in accordion component according to spec #1546
- Fixed padding in aside component according to spec #1547 
- Fixed margin in inline-button component according to spec #1550 
- Fixed background opacity in overlay card component according to spec #1552 
- Fixed icon colour in media object component according to spec #1555 
- Updated button transition to include border
- Fixed border styling in matrix component according to spec #1577 
- Fixed overlay opacity in modal component according to spec #1579 

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check that the accordion, aside, button/inline, card/overlay, media object, button, matrix and modal components match their respective specs.

## Details

Fixes #1546, fixes #1547, fixes #1550, fixes #1552, fixes #1555, fixes #1557, fixes #1577, fixes #1579, 

## Screenshots

[if relevant, include a screenshot]
